portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 1;
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var i;
+  var slides = document.getElementsByClassName("mySlides");
+  var firstSlide = document.getElementById("firstSlide");
```

Comment:
Prefer let over var https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
I left one variable (slideIndex) as var since I wanted it to be a global variable, but changed the rest to let/const

Reply:
It looks like slideIndex is in the global scope and using let wouldn't change the fact that it is global. It is still better practice to use let there. Here's a pretty good article on the differences if you're curious: https://tylermcginnis.com/var-let-const/

Reply:
I see! I will change it to let:))

Reply:
just changed it this morning!